Detectors/GlobalTracking/include/GlobalTracking/TrackCuts.h

```diff
@@ -0,0 +1,123 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// \file TrackCuts.h
+/// \brief Class to perform track cuts
+/// \author amelia.lindner@cern.ch
+
+#ifndef ALICEO2_TRACKCUTS_H
+#define ALICEO2_TRACKCUTS_H
+
+#include "Framework/Logger.h"
+#include "Framework/DataTypes.h"
+#include "ReconstructionDataFormats/Track.h"
+#include "DataFormatsGlobalTracking/RecoContainer.h"
+#include "ReconstructionDataFormats/MatchInfoTOF.h"
+#include "DataFormatsTPC/TrackTPC.h"
+#include "DataFormatsTRD/TrackTRD.h"
+#include "GlobalTracking/TrackMethods.h"
+#include "DetectorsBase/Propagator.h"
+#include "ReconstructionDataFormats/VtxTrackIndex.h"
+#include <set>
+#include <vector>
+#include "Rtypes.h"
+#include <gsl/span>
+namespace o2
+{
+using GID = o2::dataformats::GlobalTrackID;
+using GIndex = o2::dataformats::VtxTrackIndex;
+class TrackCuts
+{
+ public:
+  //////////////////////////////// O2 ////////////////////////////////
+  bool isSelected(GID trackIndex, o2::globaltracking::RecoContainer& data)
+  {
+    o2::track::TrackParCov trk;
+    auto contributorsGID = data.getSingleDetectorRefs(trackIndex);
+    auto src = trackIndex.getSource();                       // make selections depending on track source
+    if (contributorsGID[GIndex::Source::ITS].isIndexSet()) { // ITS tracks selection
+      const auto& itsTrk = data.getITSTrack(contributorsGID[GID::ITS]);
+      int ITSnClusters = itsTrk.getNClusters();
+      float ITSchi2 = itsTrk.getChi2();
+      float itsChi2NCl = ITSnClusters != 0 ? ITSchi2 / (float)ITSnClusters : 0;
+      uint8_t itsClusterMap = itsTrk.getPattern();
+      if (itsTrk.getPt() >= mMinPt && itsTrk.getPt() <= mMaxPt &&
+          itsTrk.getEta() >= mMinEta && itsTrk.getEta() <= mMaxEta &&
```

Comment:
Do you really want to apply the separate pT, eta cut on the ITS contribution? If the aim is to have a general cut on the track pT and additionally a cut on the TPC track pT, then you should remove this itsTrk.getPt() and itsTrk.getEta() checks (leaving only clusters and chi2 check) and (see below)

Reply:
I changed this now; there are now just the additional cuts in pT and eta for  TPC tracks to match the selections from MatchITSTPCQC.cxx